### PR TITLE
Add CCC blog post

### PR DESCRIPTION
**Pull request by SamLR**

### CODE REVIEW

File: _posts/2024-05-15-Bring-something-green-to-EMF.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+* If you can bring your shares and swaps in small non-glass containers or pots, that would be ideal - but, if transport space is tight, pack your swaps in plastic bags as we will have some containers available.  
+* Please label what you have brought (surprises are lovely, but …).
+* Don’t forget to keep an eye on the hydration of any growing offerings you’ve brought.
+ 
+### Bringing plants home
+
+* Be prepared to take away your new plant acquisitions (or any of your donations that haven’t been adopted) at the end of the event - so remember to save or bring clean(ish) plastic bags or pots in which to take your new acquisitions home.   
+* You could also 'post' your chosen cuttings into an empty water bottle to take home in your bags or rucksacks (you can cut open the bottle to extract them later).
+* If you’d like to make a donation to EMFCamp funds in return for adopting a new plant, seedling, cutting or some seeds, that would be much appreciated - but it’s not required.
```
Comment (jellybob) [inline _posts/2024-05-15-Bring-something-green-to-EMF.md]:
```suggestion
* If you’d like to make a donation to EMF funds in return for adopting a new plant, seedling, cutting or some seeds, that would be much appreciated - but it’s not required.
```

File: _posts/2024-05-15-Bring-something-green-to-EMF.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+![A plant growing in a glass beaker held up against the light; ready to go to EMF](../../../../images/2024-05-ccc-pot-plant.png "A plant for EMF")
+
+## The practical details
+
+For the CCC to work we both need you to bring plants & take them home with you
+
+### Bringing plants to Electromagnetic Field 2024
+
+* If you can bring your shares and swaps in small non-glass containers or pots, that would be ideal - but, if transport space is tight, pack your swaps in plastic bags as we will have some containers available.  
+* Please label what you have brought (surprises are lovely, but …).
```
Comment (jellybob) [inline _posts/2024-05-15-Bring-something-green-to-EMF.md]:
```suggestion
* Please label what you have brought (surprises are lovely, but…).
```

File: _posts/2024-05-15-Bring-something-green-to-EMF.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+* Don’t forget to keep an eye on the hydration of any growing offerings you’ve brought.
```
Comment (jellybob) [inline _posts/2024-05-15-Bring-something-green-to-EMF.md]: I’m not entirely clear what this is asking. Should people make they’re hydrated in transit, or do they also need to check in now and again on site?

File: _posts/2024-05-15-Bring-something-green-to-EMF.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+ 
+### Bringing plants home
+
+* Be prepared to take away your new plant acquisitions (or any of your donations that haven’t been adopted) at the end of the event - so remember to save or bring clean(ish) plastic bags or pots in which to take your new acquisitions home.   
+* You could also 'post' your chosen cuttings into an empty water bottle to take home in your bags or rucksacks (you can cut open the bottle to extract them later).
```
Comment (jellybob) [inline _posts/2024-05-15-Bring-something-green-to-EMF.md]:
```suggestion
* You could also post your chosen cuttings into an empty water bottle to take home in your bags or rucksacks (you can cut open the bottle to extract them later).
```

File: _posts/2024-05-15-Bring-something-green-to-EMF.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+* If you’d like to make a donation to EMFCamp funds in return for adopting a new plant, seedling, cutting or some seeds, that would be much appreciated - but it’s not required.
+
+**Important**
+And we’re really very sorry about this but, due to import/export regulations, please don’t take any plants or seeds out of the UK - but we hope our international visitors can take home some green-fingered inspiration and enthusiasm instead!
```
Comment (jellybob) [inline _posts/2024-05-15-Bring-something-green-to-EMF.md]:
```suggestion
We’re really very sorry about this but, due to import/export regulations, please don’t take any plants or seeds out of the UK - we hope our international visitors can take home some green-fingered inspiration and enthusiasm instead!
```

File: _posts/2024-05-15-Bring-something-green-to-EMF.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+## Non-plant contributions
+And for those creative makers who don’t grow, you could contribute ….. 
+* Some  small light-weight hanging features to complement the amazing creations people are bringing to decorate the Lounge. 
+* Any/all of macrame, woven, canvas, leather, fabric or wooden pot hangers & mobiles.
+* Inspirational ideas for growers or wildlife enthusiasts that you could bring and put on display.
+
+Displays can be electrical, mechanical or solar powered (but unfortunately we won’t be able to demonstrate anything hydro powered as there is no water supply in the tent).
+
+Ending this post by saying that we’re jumping up and down with enthusiastic anticipation of seeing a wide variety of green and creative contributions to Electromagnetic Field 2024!
```
Comment (jellybob) [inline _posts/2024-05-15-Bring-something-green-to-EMF.md]:
```suggestion
We’re jumping up and down with enthusiastic anticipation of seeing a wide variety of green and creative contributions to Electromagnetic Field 2024!
```

File: _posts/2024-05-15-Bring-something-green-to-EMF.md (hunk 3 above)
Comment (SamLR) [reply _posts/2024-05-15-Bring-something-green-to-EMF.md]: The latter I think (tweak incoming)

**Review by jellybob — APPROVED** [review]

**Pull request merged**